source/MapPanel.cpp

```diff
@@ -1143,16 +1144,20 @@ void MapPanel::DrawTooltips()
 
 
 
-void MapPanel::DrawPointer(const System *system, Angle &angle, const Color &color, bool bigger)
+void MapPanel::DrawPointer(const System *system, int &systemCount, const Color &color, bool bigger)
 {
-	DrawPointer(Zoom() * (system->Position() + center), angle, color, true, bigger);
+	DrawPointer(Zoom() * (system->Position() + center), systemCount, color, true, bigger);
 }
 
 
 
-void MapPanel::DrawPointer(Point position, Angle &angle, const Color &color, bool drawBack, bool bigger)
+void MapPanel::DrawPointer(Point position, int &systemCount, const Color &color, bool drawBack, bool bigger)
 {
-	angle += Angle(30.);
+	++systemCount;
+	Angle angle = Angle(30.*systemCount);
+	if(systemCount>12) {
+    	return;
```

Comment:
There still is an issue with spaces instead of tabs here.

Reply:
Thanks for spotting it! Is it OK now?

Reply:
Yes, it is OK now.